docs/make.jl

```diff
@@ -57,6 +57,7 @@ makedocs(
                 "Tree mesh" => joinpath("meshes", "tree_mesh.md"),
                 "Structured mesh" => joinpath("meshes", "structured_mesh.md"),
                 "Unstructured mesh" => joinpath("meshes", "unstructured_quad_mesh.md"),
+                "Simplicial mesh" => joinpath("meshes", "mesh_data_meshes.md"),
                 "P4est-based mesh" => joinpath("meshes", "p4est_mesh.md"),
```

Comment:
```suggestion
                "P4est-based mesh" => joinpath("meshes", "p4est_mesh.md"),
                "Simplicial mesh" => joinpath("meshes", "mesh_data_meshes.md"),
```
I think it might be better to sort the meshes by element type